CHANGELOG.md

```diff
@@ -1,3 +1,123 @@
+# [2.5.0-alpha.11](https://github.com/ar-io/ar-io-sdk/compare/v2.5.0-alpha.10...v2.5.0-alpha.11) (2024-11-21)
```

Comment:
chore: can scrub alpha logs

Reply:
ty for the reminder! [015eca1](https://github.com/ar-io/ar-io-sdk/pull/261/commits/015eca108aa82500d8c39b493f34ea6e1c9e86b6)